packages/babel-node/src/_babel-node.js

```diff
@@ -70,6 +75,7 @@ register({
   presets: program.presets,
   configFile: program.configFile,
   envName: program.envName,
+  rootMode: program.rootMode,
```

Comment:
Would you be up for moving this options object into a variable and adding a loop like https://github.com/babel/babel/blob/1b8d664bbe1976ae294a0c74c3a24b7dbb7e33b5/packages/babel-cli/src/babel/options.js#L257 Otherwise this will run into the same issue the CLI did that the comment https://github.com/babel/babel/blob/1b8d664bbe1976ae294a0c74c3a24b7dbb7e33b5/packages/babel-cli/src/babel/options.js#L253 describes.

Reply:
Sure.  Do you want me to move it to an `options.js` as with `@babel/cli`?

Reply:
Nah, just moving it above the `register` call is fine.